shortlinks.csv

```diff
@@ -44,7 +44,7 @@ finances,https://docs.google.com/spreadsheets/d/1ZHCGpTCA3YNvYI4aW1dU9UKsCeMyAcR
 facebook,https://www.facebook.com/hyphacoop/
 fb,https://www.facebook.com/hyphacoop/
 photos,https://drive.google.com/drive/u/0/folders/105SjE7YSlaIyhJvSw7aMnkfKjUtaXNcr
-meetings,https://hackmd.io/c/v6ygUjDeRKaSeoi3BqpSMg
+meetings,https://organizing.hypha.coop/
```

Comment:
@dcwalk suggests `meetings.hypha.coop` because it's organizing is very "handbook" overlap

```suggestion
meetings,https://meetings.hypha.coop/
```